lib/ProductOptions.php

```diff
@@ -5,8 +5,9 @@
 class ProductOptions
 {
     public const PLAN_ID = 'configoption1';
-    public const SERVERS = 'configoption2';
-    public const WEBSITES = 'configoption3';
+    public const DOMAIN = 'configoption2';
+    public const SERVERS = 'configoption3';
+    public const WEBSITES = 'configoption4';
```

Comment:
Is this not a BC? Because `SERVERS` changes from `configoption2` to `configoption3` and `WEBSITES` from `configoption3` to `configoption4`?

Reply:
I'm also interested in this =)

Reply:
Yes, BC break but this time we will make an exception